README.md

```diff
@@ -76,6 +76,26 @@ store
   });
 ```
 
+## Why use this?
+
+Having used and followed the progression of Redux and the Elm Architecture, and
+after trying other effect patterns for Redux, we came to the following
+conclusion:
+
+> Synchronous state transitions caused by returning a new state from the reducer
+> in response to an action are just one of all possible effects an action can
+> have on application state.
+
+Many of the other possible methods for handling effects in Redux which focus on
+action-creators guide the user either explicitly or implicitly to understand
+asynchronous effects as a significantly different concept than reducer state
+transitions simply because they are asynchronous. With `redux-loop`, the reducer
+doesn't just decides what happens _*now*_ due to a particular action, it decides
+what happens _*next*_. All of the behavior of your application can be traced
+through one place, and that behavior can be easily broken apart and composed
+back together. This is one of the most powerful features of the Elm
+Architecture, and with `redux-loop` it can be a feature of Redux as well.
```

Comment:
s/Architecture/architecture. I'd also replace "can be" with "is". Just a bit more affirmative, that way.